themes/default/content/docs/intro/console/organizations.md

```diff
@@ -125,6 +125,30 @@ If you need help configuring or would like us to officially support another SAML
 provider,
 please [contact us]({{< relref "/about#contact-us" >}}).
 
+## Changing Membership Requirements
+
+Every organization is backed by an identity. The identity governs the membership to your org.
+When you add a new org to Pulumi, it uses the Pulumi identity, by default. This means the membership is
+_only_ governed by a user having a Pulumi account and no additional identity requirements are placed on members.
+
+{{% notes %}}
+Regardless of the identity your org uses, org membership is always invite-only. Only the creator of the org gets automatic access. Everyone else must be invited by an admin to be admitted into the org.
+{{% /notes %}}
+
+However, if you want your org to mirror an org on a third party identity service such as GitHub, GitLab, or Bitbucket, you can
+change the backing identity. Enterprises can also choose SAML as the backing identity provider for an org as you read
+in the previous section. Changing your org's backing identity essentially changes the membership requirements you place
```

Comment:
Perhaps "as discussed in []()" with a link to the anchor?  (Just in case this content gets reorganized in the future).